src/controllers/ArtsController.js

```diff
@@ -63,15 +63,17 @@ class ArtsController {
           categoryId,
           featuredImg: mediaFilesArray[0].url
             || process.env.DEFAULT_ARTICLE_IMAGE,
-          status: defaultStatus
+          status: defaultStatus,
+          visited: 0
```

Comment:
You can just set a default value for this in your migration/model file.

Reply:
fixed